frontend/packages/dev-console/src/components/pipelines/resource-overview/DynamicResourceLinkList.tsx

```diff
@@ -13,21 +14,27 @@ export type ResourceModelLink = {
 type DynamicResourceLinkListProps = {
   links: ResourceModelLink[];
   namespace: string;
-  title: string;
+  title?: string;
+  addSpaceBelow?: boolean;
 };
 
 const DynamicResourceLinkList: React.FC<DynamicResourceLinkListProps> = ({
   links = [],
   namespace,
   title,
+  addSpaceBelow = true,
 }) => {
   if (links.length === 0) {
     return null;
   }
   return (
-    <div className="odc-dynamic-resource-link-list">
+    <div
+      className={classNames({
+        'odc-dynamic-resource-link-list': addSpaceBelow,
+      })}
```

Comment:
Technically [BEM](http://getbem.com/introduction/) wants us to use modifiers in this case. Suggest this:
```suggestion
      className={classNames('odc-dynamic-resource-link-list', {
        'odc-dynamic-resource-link-list--addSpaceBelow': addSpaceBelow,
      })}
```